.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish EternalCombat API
+on:
+  push:
+    branches: [ master ]
```

Comment:
Please move to on release, why do you want to publish eternal-combat api for all pushes?